scripts/run_all_checks.py

```diff
@@ -0,0 +1,63 @@
+import json
```

Comment:
It makes sense to me to place scripts that I use for a repo in a scripts folder, as to not clutter the root any more than it usually is with project-wide config files. It also makes them easier to find.
But that's just how I do it in *my* projects. I can move it to wherever works best for typeshed.

I'll change the name, it's more concise. I don't tend to mind long names because of tab-completion.